agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -474,6 +474,10 @@ func TestV3TaskEndpointHostNetworkMode(t *testing.T) {
 	testV3TaskEndpoint(t, "v3-task-endpoint-validator", "v3-task-endpoint-validator", "host", "ecs-functional-tests-v3-task-endpoint-validator")
 }
 
+func TestV3TaskEndpointTags(t *testing.T) {
```

Comment:
Since you also add the V2 endpoint, test cases need to be added accordingly.

Reply:
I've edited the V2 functional test to check for Tags now

Reply:
I think there is value in testing v2 metadata without long arn format too. I would suggest adding a new test for tags.

Reply:
TestTaskMetadataValidator (below) checks the v2 endpoint. For both, we need long arn format because Tags require it. Not having long Arn will throw an error from the ECS API and be reported in Agent logs.